dts/bindings/sensor/bosch,bmp180-i2c.yaml

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Chris Ruehl
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Bosch BMP180 pressure sensor accessed through I2C bus
+
+compatible: "bosch,bmp180"
```

Comment:
Since it can only be on the i2c bus anyways, combine the two bindings

Reply:
agree